### PR TITLE
Add tests fro datepicker

### PR DESCRIPTION
**Pull request by Sartonon**

### CODE REVIEW

File: src/hooks/useDropdownKeyboardNavigation.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const useDropdownKeyboardNavigation = ({
           break;
         case 'ArrowDown':
           if (isInitialNavigation && typeof initialFocusedIndex === 'number') {
-            focusOption('down', initialFocusedIndex);
+            focusOption('down', initialFocusedIndex - 1);
```
Comment (jorilindell) [inline src/hooks/useDropdownKeyboardNavigation.tsx:71]: What is going to happen if initialFocusedIndex is 0?

File: src/common/components/datepicker/TimesList.tsx [hunk 2]
```diff
@@ -111,7 +111,7 @@ const TimeItem: React.FC<TimeItemProps> = ({
   // scroll focused element into view
   React.useEffect(() => {
     if (focused) {
-      buttonRef.current?.scrollIntoView({
+      buttonRef.current?.scrollIntoView?.({
```
Comment (jorilindell) [inline src/common/components/datepicker/TimesList.tsx:114]: I'm only curious here, but what is the reason for ?-mark in scrollIntoView?

Comment (Sartonon) [reply src/common/components/datepicker/TimesList.tsx:114]: jsdom seems to not support that one and errors out. Maybe need add comment about that?

File: src/hooks/useDropdownKeyboardNavigation.tsx (hunk 1 above)
Comment (Sartonon) [reply src/hooks/useDropdownKeyboardNavigation.tsx:71]: First down key will focus the first item. At least seems to work correctly for times list and language selector

Comment (jorilindell) [reply src/hooks/useDropdownKeyboardNavigation.tsx:71]: Ok, great

File: src/common/components/datepicker/TimesList.tsx (hunk 2 above)
Comment (jorilindell) [reply src/common/components/datepicker/TimesList.tsx:114]: That's something that I was expecting. Maybe comment could be helpful. It was not obvious to me when I saw this at first time.

**Pull request merged**